data-access/Models/UserModels.cs

```diff
@@ -36,90 +36,47 @@ public class UserEditionPermissions
 		public bool Locked { get; set; }
 	}
 
-	public class UserInfo
+	public class EditionUserInfo
```

Comment:
I don't like this name. It's too literal. The name should explain the purpose of this class, not what it holds. Unfortunately, I don't have a better name, so let's just keep this one.